Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-16LE
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 13a07e7168e0dff29f282a9a5af60de8b73823aa)
+++ b/.gitignore	(date 1768796043968)
@@ -1,5 +1,6 @@
 node_modules/
 dist/
+.idea
 /node_modules/.bin/asar
 /node_modules/.bin/asar.cmd
 /node_modules/.bin/asar.ps1
